invisible_cities/cities/detsim_get_psf_test.py

```diff
@@ -0,0 +1,88 @@
+import os
+import numpy  as np
+import tables as tb
+import pandas as pd
+
+from pytest import fixture
+
+from invisible_cities.cities.detsim_get_psf import *
+
+####################################
+######### CREATE XY FUNC ###########
+####################################
+@fixture
+def extent_and_function():
+    xmin, xmax, dx = 12, 100, 3.
+    ymin, ymax, dy =-50, 50, 13.
+
+    xaxis = (xmin, xmax, dx)
+    yaxis = (ymin, ymax, dy)
+
+    func = lambda x, y: - x**2
```

Comment:
why does it have y as an argument if it only uses x?

Reply:
No reason for that, the function `func` is just a x, y, z (new code adds the z) for testing `create_xyz_function`. In the case it is there, just a x dependence is assumed.

Reply:
It's a lambda function in a fixture that takes 3 arguments and only uses one?! Why? If you want it as a place holder for a more complex function which will use the other arguments write the full function and make it return `-x**2` if y and z are 0 or something like that. If it's always just going to give `-x**2` than just define it with one parameter `x`

Reply:
This is just a matter of taste. This function can be modified to whatever xyz function you want. Note that its aim is in `test_create_xyz_function`. I changed it with explicit x, y, z dependence inside, maybe now it look better for you.

Reply:
Well no, but thanks for changing it so that y and z are relevant